functions/messenger.js

```diff
@@ -107,6 +108,7 @@ class ChatwootMessenger {
           this.inboxSourceId = this._findSourceId(user.contact_inboxes);
         }
       } catch (error) {
+        await reportError(error);
         throw new CreateContactError(error);
```

Comment:
Do we need to throw an error if we are manually capturing it with Sentry? Sentry will capture this `throw` here too.

Also, I'm not sure if this is causing other issues with Netlify, seems like the function retries when there's an error? I'm not 100% sure of this but may explain why we are getting duplicated messages on Chatwoot